### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by David-Lor** [jwalton/gh-find-current-pr#12]
Using the following workflow:

```yaml
on:
  pull_request:
    branches:
      - main
    types:
      - closed

name: Tag on merged PR

jobs:
  tag:
    name: Tag merged MR
    if: github.event.pull_request.merged == true
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v1
      - name: Get PR description
        uses: jwalton/gh-find-current-pr@v1
        id: findPr
      - name: Echo PR data
        run: echo "Number = ${number} | Title = ${title} | Body = ${body}"
        env:
          number: ${{ steps.findPr.outputs.pr }}
          title: ${{ steps.findPr.outputs.title }}
          body: ${{ steps.findPr.outputs.body }}
```

The `steps.findPr.outputs.*` fields return empty.

If I modify the workflow to use the example given in this repository readme, the step does not run, as if it was not triggered by a merged PR. I'm working on a Private repository (for testing purposes), maybe this is not possible with this Action? I tried using the GITHUB_TOKEN, as I saw on the Action source that it was an optional parameter.

```yaml
on:
  push:
    branches:
      - main

name: Tag on merged PR

jobs:
  tag:
    name: Tag merged MR
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v1
      # Find the PR associated with this push, if there is one.
      - uses: jwalton/gh-find-current-pr@v1
        id: findPr
        env:  # tried with "env" and "with"
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      # This will echo "Your PR is 7", or be skipped if there is no current PR.
      - run: echo "Your PR is ${PR}"
        if: success() && steps.findPr.outputs.number
        env:
          PR: ${{ steps.findPr.outputs.pr }}
```

**Comment by rob-baldwin** [comment]
@David-Lor try:

```
- uses: jwalton/gh-find-current-pr@v1.0.2
  id: findPr
  with:
    github-token: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by dudicoco** [comment]
does not work for me as well.

@jwalton was this action tested with private repositories?

**Comment by jwalton** [comment]
Yes.  I developed this at my previous job, and was using it exclusively on private repos.  Afraid I don't have any repos I'm actively using it on at the moment, though.

The [implementation](https://github.com/jwalton/gh-find-current-pr/blob/de950fc8692ca0076b5e0fb9061816809d1715d6/main.js#L10-L16) is pretty straightforward - we just use the Github API to find all PRs associated with the commit, and then choose the first open one.

**Comment by dudicoco** [comment]
@jwalton thanks

I believe the problem might be that you are looking only for open PRs, while PRs that were merged are considered closed.

**Comment by jwalton** [comment]
Ah, yes it won’t find a merged PR.  I can add an option to allow closed PRs.

**Comment by David-Lor** [comment]
@jwalton Just tried it, working perfectly fine now!

**Comment by jwalton** [comment]
Released as v1.2.0.
